aptos-move/framework/aptos-framework/sources/configs/gas_schedule.move

```diff
@@ -23,16 +31,27 @@ module aptos_framework::gas_schedule {
         entries: vector<GasEntry>
     }
 
+    /// Called during a module upgrade.
```

Comment:
You can't call this in the same proposal execution transaction as the updated module code would not be available yet.

Reply:
True. I'll fix the comment.